lightwood/encoder/numeric/numeric.py

```diff
@@ -80,7 +94,13 @@ def encode(self, data):
 
         return torch.Tensor(ret)
 
-    def decode(self, encoded_values, decode_log=None) -> list:
+    def decode(self, encoded_values: Union[List[Number], torch.Tensor], decode_log: bool = None) -> list:
+        """
+        :param encoded_values: The encoded values to decode into single numbers
+        :param decode_log: Wether to decode the ``log`` or ``linear`` part of the representation, since the encoded vector contains both a log and a linear part
+
+        :returns: A number
```

Comment:
can we say "returns the number expected from decoding"

Reply:
Not sure what `expected` represents in that context.